hivemq4/dns-image/Dockerfile

```diff
@@ -14,14 +14,16 @@ ENV HIVEMQ_CLUSTER_TRANSPORT_TYPE UDP
 
 COPY config-dns.xml /opt/hivemq/conf/config.xml
 
-RUN mkdir /opt/hivemq/extensions/hivemq-dns-cluster-discovery
-COPY hivemq-dns-cluster-discovery/* /opt/hivemq/extensions/hivemq-dns-cluster-discovery/
+RUN curl -L https://github.com/hivemq/hivemq-dns-cluster-discovery-extension/releases/download/4.1.0/hivemq-dns-cluster-discovery-4.1.0.zip -o /opt/hivemq/extensions/dns-discovery.zip \
```

Comment:
Quick thought, for HiveMQ we specify the version as ARG 
https://github.com/hivemq/hivemq4-docker-images/blob/master/hivemq4/base-image/Dockerfile#L3
and then download it later so the version can be overridden in the build process
https://github.com/hivemq/hivemq4-docker-images/blob/master/hivemq4/base-image/Dockerfile#L40

To align the behaviour it might make sense to to the same here - what's your thoughts on this?